tests_and_analysis/performance_benchmarking/utils.py

```diff
@@ -42,3 +42,16 @@ def get_qpts() -> np.ndarray:
     """
     qpts_npy_file = os.path.join(get_data_path(), "qpts_10000.npy")
     return np.load(qpts_npy_file)
+
+
+def get_san_storage() -> str:
+    """
+    Returns
+    -------
+    str
+        The server location of the SAN storage.
+    """
+    return (
+        r'\\isis.cclrc.ac.uk\Shares\PACE_Project_Tool_Source'
+        r'\euphonic_performance_benchmarking'
+    )
```

Comment:
In Horace they've tried to keep the SAN storage location out of Github - we should probably do the same. It's convenient to have it hardcoded in but not essential - I have to manually put in my Jenkins user/token anyway so putting in the storage location too isn't too much trouble

Reply:
My bad, removed